docs/development.md

```diff
@@ -0,0 +1,181 @@
+
+Then, to develop `saleor-app-checkout` app locally, you need to create a tunnel for it. The tunnel will enable you to display it within your Saleor Dashboard.
+
+If you want to read more about Saleor Apps architecture, see [the documentation](https://docs.saleor.io/docs/3.x/developer/extending/apps/key-concepts).
+
+To create a tunnel, run:
+
+```bash
+cd apps/saleor-app-checkout && npx saleor app tunnel 3001
+```
+
+> Note: The process needs to be running in the background. Please don't kill it during the development.
+
+<!-- is the bit below correct? you have to filter something to run it for all the apps? -->
+<!-- 
+To run the development server for all the apps, use the following command:
+
+```
+pnpm run dev --filter=saleor-app-checkout...
+```
+ -->
+#### [`checkout-storefront`](../packages/checkout-storefront/README.md)
+
+The URL structure for `checkout-storefront` is `localhost:3002?checkout=<ID>`, which means it requires a checkout id to work correctly. You don't have to supply it manually; it will be provided when transitioning from the cart to checkout.
+
+### Building
+
+To build all apps and packages, run the following command:
+
+```
+pnpm run build
+```
+
+You can also build a specific app or package by running this command:
+
+```bash
+pnpm run build --filter=saleor-app-checkout
+```
+
+In this example, we'll only build `apps/saleor-app-checkout`
+
+### Ports
+
+- apps/react-storefront: **http://localhost:3000**
+- apps/saleor-app-checkout: **http://localhost:3001**
+- packages/checkout-storefront: **http://localhost:3002**
+- packages/ui-kit storybook: **http://localhost:6006**
+
+## Other
+
+### GraphQL Schema
+
+To generate GraphQL code based on the latest schema from Saleor instance defined in `SALEOR_API_URL` env variable, run this command:
+
+```
+pnpm run generate
+```
+
+You need to run this command after each change in the `*.graphql` files.
+  
+### Code formatting
+
+Before committing the code, Git pre-hooks will check staged changes for following the code styles. If you would like to format the code by yourself, run the command:
+
+```bash
+pnpm lint
+```
+  
+### Configuration
+
+Instructions how to configure the application (e.g. change the GraphQL API URL) can be found [here](apps/storefront/docs/configuration.md).
+
+### Env variables
+
+Change environment variables inside `.env` file:
+
+- `SALEOR_API_URL` — GraphQL endpoint of your Saleor
+
+  Example:
+
+  ```
+  https://my-env.eu.saleor.cloud/graphql/
+  ```
+
+  > To run sandbox Saleor environment in [Saleor Cloud](https://cloud.saleor.io/) use this command:
+  >
+  > ```bash
+  > npx saleor project create && npx saleor environment create
+  > ```
+  >
+  > You can also run Saleor locally. See [Saleor docs](https://docs.saleor.io/docs/3.x/developer/installation) for more instructions
```

Comment:
I think we can drop this bit? Since we're suggesting to use `saleor storefront create` in the main README.

We can leave the link to docs about installation though